src/compass/utils/runconfig.py

```diff
@@ -453,5 +458,5 @@ def to_yaml(self):
         '''Dump runconfig as string to sys.stdout
         '''
         self_as_dict = self.as_dict()
-        yaml = YAML(typ='safe')
-        yaml.dump(self_as_dict, sys.stdout)
+        yaml_obj = YAML(typ='safe')
```

Comment:
just checking- any reason for this change?

Reply:
Not necessary, but appended `_obj` to avoid conflict with `import yaml` up top.